### ISSUE TITLE
Add AT file check

### ISSUE THREAD

**Issue by Raycoms** [ldtteam/OperaPublicaCreator#6]

**Comment by Nightenom** [comment]
should be just if (isFeatureDisabled("at_file")) around <https://github.com/ldtteam/OperaPublicaCreator/blob/ng7/gradle/mod.gradle#L554>

**Comment by marchermans** [comment]
Yes but only on the ng7 branch and with the feature also checking if the default file exists to maintain backwards compat
